### PR TITLE
Add info on lepton-cli to the reference manual

### PR DESCRIPTION
**Pull request by vzh**
Affects #284.

### CODE REVIEW

File: docs/manual/lepton-eda-fonts.texi [hunk 1]
```diff
@@ -0,0 +1,104 @@
+@node Lepton EDA fonts, Topic Index, gEDA file format, Top
+@appendix Lepton EDA fonts
+@cindex Lepton EDA fonts
+@cindex Fonts configuration in Lepton EDA
+
+On Linux, font configuring is handled by @code{Fontconfig}, and modern
```
Comment (graahnul-grom) [inline docs/manual/lepton-eda-fonts.texi:6]: (and many other UNIX-like systems)

File: docs/manual/lepton-eda-fonts.texi [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+systems (particularly @code{pango} and @code{cairo}) rely on its
+library in looking up any font specified.  You can provide the
```
Comment (graahnul-grom) [inline docs/manual/lepton-eda-fonts.texi:8]: "rely on *this* library"?

File: docs/manual/lepton-eda-fonts.texi [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+SemiExpanded
+
+Expanded
+
+ExtraExpanded
+
+UltraExpanded
+
+@end multitable
```
Comment (graahnul-grom) [inline docs/manual/lepton-eda-fonts.texi:103]: Could the table line spacing be expanded a little?
Or is it drawn incorrectly on my system only (I can't tell one table row from the other)?

File: docs/manual/lepton-cli-shell.texi [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+configuration files:
+
+@example
+lepton-cli shell -c \
+  "(use-modules (lepton os)) (display (sys-config-dirs))"
+@print{} (/etc/xdg/lepton-eda /usr/local/share/lepton-eda)
+@end example
+
+Hopefully, you don't need this for any of your project's directories
+;-) But who knows...  If so, try to compose a useful command yourself.
```
Comment (graahnul-grom) [inline docs/manual/lepton-cli-shell.texi:63]: Would it be appropriate to add a link to the [scripting examples](https://github.com/lepton-eda/lepton-eda/wiki/Scripting-examples) wiki page here?

**Review by graahnul-grom — APPROVED** [review]
Very useful examples and hints for the `lepton-cli`
`export` and `config` commands.

File: docs/manual/lepton-eda-fonts.texi (hunk 1 above)
Comment (vzh) [reply docs/manual/lepton-eda-fonts.texi]: Fixed, thanks.

File: docs/manual/lepton-eda-fonts.texi (hunk 2 above)
Comment (vzh) [reply docs/manual/lepton-eda-fonts.texi]: Fixed as well.

File: docs/manual/lepton-eda-fonts.texi (hunk 3 above)
Comment (vzh) [reply docs/manual/lepton-eda-fonts.texi:103]: Please look at the last recent commit.  Unfortunately, I cannot change the style of just one table, IIUC. Therefore other tables affected, including those representing menus with links to subsections in info manual.

File: docs/manual/lepton-cli-shell.texi (hunk 4 above)
Comment (vzh) [reply docs/manual/lepton-cli-shell.texi:63]: I think it should go to another section/chapter, something like *Extensions in Scheme*.

File: docs/manual/lepton-eda-fonts.texi (hunk 3 above)
Comment (graahnul-grom) [reply docs/manual/lepton-eda-fonts.texi:103]: It looks much better to me. Thank you.

File: docs/manual/lepton-cli-shell.texi (hunk 4 above)
Comment (graahnul-grom) [reply docs/manual/lepton-cli-shell.texi:63]: Excellent idea! One thing: I beg you, do not assume all our
users are DBCS-ninjas. The simpler - the better.
BTW, "3.5 Deprecated settings" is empty (?).

**Pull request merged**